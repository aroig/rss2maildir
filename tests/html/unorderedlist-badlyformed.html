<ul>
    <li>This is an item
    <li>And a second
    <li>And a third
    </li></li></li>
</ul>
