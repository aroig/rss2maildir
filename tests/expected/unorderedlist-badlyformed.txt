
 * This is an item
 * And a second
 * And a third
